astronomer/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -27,7 +41,7 @@ async def describe_transform_job_async(self, job_name: str) -> Dict[str, Union[s
             response: Dict[str, Any] = await client.describe_transform_job(TransformJobName=job_name)
             return response
 
-    async def describe_processing_job_async(self, job_name: str) -> Dict[str, Union[str, List[str]]]:
+    async def describe_processing_job_async(self, job_name: str) -> Dict[str, Any]:
```

Comment:
not your change but don't like putting "async" in the function name since the definition is clear and we don't have conflicting function name

Reply:
what if we have the same name in the sync version and later we merge this async with OSS ? there is function `describe_processing_job` in the sync version to differentiate `async` was added